jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
           "subdir": "grafana"
         }
       },
-      "version": "master"
+      "version": "release-0.1"
```

Comment:
I didn't cut another release branch here as there was no recent bump here in kube-prometheus and the commit in the lock file still points to the `release-0.1` commit. let me know if you feel i should bump to latest master here too, and cut a `release-0.2` in `kubernetes-grafana`.

Reply:
The current line is fine by me